server/models/user.js

```diff
@@ -14,8 +14,8 @@ const userSchema = new Schema({
   groups: [{ type: Schema.Types.ObjectId, ref: 'Group' }],
   usersIFollow: [{ type: Schema.Types.ObjectId, ref: 'User' }],
   usersFollowingMe: [{ type: Schema.Types.ObjectId, ref: 'User' }],
-  exploreNumber: Number,
-  numExplorations: Number,
+  exploreNumber: { type: Number, default: 0 },
+  numExplorations: { type: Number, default: 20 },
```

Comment:
So this is just assuming that explore won't serve anything until they've annotated 20 articles? Or assuming that we will be able to pull in 20 scores from FB?